src/util/updateActiveNavItem.js

```diff
@@ -11,7 +11,7 @@ export default function updateActiveNavItem(activeSection, nav) {
   }
 
   const newActive = nav.querySelector(
-    '[data-sn-section=' + activeSection.id + ']'
+    '[data-sn-section="' + activeSection.id + '"]'
```

Comment:
Not sure why I didn't think to make this a template string, mind updating it for easier reading and less quote marks?